src/js/pages/Login/components/WelcomeComponent.js

```diff
@@ -106,12 +103,10 @@ const  WelcomePage = styled.div`
 
 WelcomePage.displayName='WelcomePage';
 
-const Icon= styled.img`
-    height: 8vw;
-    width: 8vw;
-    min-height: 100px;
-    min-width: 100px;
-  `;
+const Icon = styled.img`
+  height: 8vw;
+  width: 9.5vw;
+`;
```

Comment:
i see in some places we've removed some min-width/min-height styles. i know that's probably bc the min-style meant that it would be too big for some screen sizes, but i think that we need some stopping point for the resizing, otherwise it could become potentially to small depending on how the user resizes the screen? what do you think?

Reply:
I understand your concern. Do you think we need to specify min values in pixels? If so, I think they will look very different on different screens. For example, on desktop (1920x1080) a button looked normal, but on a mobile device (800x400) a button was very big and broke the UI. So I'm not sure what the best approach to handle that.